README.md

```diff
@@ -1,51 +1,307 @@
+2. [DSPipeline Component Overview](#dspipeline-component-overview)
+3. [Using a DSPipeline](#using-a-dspipeline)
+   1. [Using the Graphical UI](#using-the-graphical-ui)
+   2. [Using the API](#using-the-api)
+4. [Cleanup](#cleanup)
+   1. [Cleanup ODH Installation](#cleanup-odh-installation)
+   2. [Cleanup Standalone Installation](#cleanup-standalone-installation)
+5. [Run tests](#run-tests)
 
 # Quickstart
 
-Deploy the operator
+To get started you will first need to satisfy the following pre-requisites:
+
+## Pre-requisites
+1. An OpenShift cluster that is 4.9 or higher.
+2. You will need to be logged into this cluster as [cluster admin] via [oc client].
+3. The OpenShift Cluster must have OpenShift Pipelines 1.7.2 or higher installed. Instructions [here][OCP Pipelines Operator].
+4. Based on installation type you will need one of the following:
+   1. For Standalone method: You will need to have [Kustomize] version 4.5+ installed
+   2. For ODH method: The Open Data Hub operator needs to be installed. You can install it via [OperatorHub][installodh].
+
+## Deploy the Operator via ODH
+
+On a cluster with ODH installed, create a namespace where you would like to install DSPO: 
+
+```bash
+DSPO_NS=data-science-pipelines-operator
+oc new-project ${DSPO_NS}
+```
+
+Then deploy the following `KfDef` into the namespace created above:
+
+```bash
+cat <<EOF | oc apply -f -
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+   name: data-science-pipelines-operator
+   namespace: ${DSPO_NS}
+spec:
+   applications:
+      - kustomizeConfig:
+           repoRef:
+              name: manifests
+              path: data-science-pipelines-operator/
+        name: data-science-pipelines-operator
+   repos:
+      - name: manifests
+        uri: "https://github.com/opendatahub-io/odh-manifests/tarball/master"
+EOF
+```
+
+Confirm the pods are successfully deployed and reach running state:
+
+```bash
+oc get pods -n ${DSPO_NS}
+```
+
+Once all pods are ready, we can proceed to deploying the first Data Science Pipelines (DSP) instance. Instructions
+[here](#deploy-dsp-instance).
+
+## Deploy the Operator standalone
+
+First clone this repository: 
+
+```bash
+WORKGING_DIR=$(mktemp -d)
+git clone https://github.com/opendatahub-io/data-science-pipelines-operator.git ${WORKGING_DIR}
+```
+If you already have the repository cloned, set `WORKGING_DIR=` to its absolute location accordingly.
+
+DSPO can be installed in any namespace, we'll deploy it in the following namespace, you may update this environment 
+variable accordingly.
+
+```bash
+DSPO_NS=data-science-pipelines-operator
+
+# Create the namespace if it doesn't already exist
+oc new-project ${DSPO_NS}
+```
+
+Now we will navigate to the DSPO manifests then build and deploy them to this namespace.
+
 ```bash
-oc new-project data-science-pipelines-operator
-cd ${REPO}/config/default
+cd ${WORKGING_DIR}/config/base
 kustomize build . | oc apply -f -
```

Comment:
should we use `make` instead? `make deploy`? not sure if it needs to be tweaked though.

```
.PHONY: deploy
deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
	$(KUSTOMIZE) build config/default | kubectl apply -f -

```

Reply:
great idea, I updated the makefile and instructions accordingly